pkg/storage/tsdb/index_cache.go

```diff
@@ -51,7 +51,7 @@ func (cfg *IndexCacheConfig) RegisterFlags(f *flag.FlagSet) {
 }
 
 func (cfg *IndexCacheConfig) RegisterFlagsWithPrefix(f *flag.FlagSet, prefix string) {
-	f.StringVar(&cfg.Backend, prefix+"backend", IndexCacheBackendDefault, fmt.Sprintf("The index cache backend type. Supported values: %s.", strings.Join(supportedIndexCacheBackends, ", ")))
+	f.StringVar(&cfg.Backend, prefix+"backend", IndexCacheBackendDefault, fmt.Sprintf("Comma separated ordered list of index cache backend type. Supported values: %s.", strings.Join(supportedIndexCacheBackends, ", ")))
```

Comment:
Should there be a blurb about multi-level caching?

Reply:
better?